storage/rocksdb/src/block.rs

```diff
@@ -1,19 +1,21 @@
 use {
     crate::{
-        generic::{FromKey, FromValue, ToKey, ToValue},
+        generic::{FromValue, ToKey, ToValue},
         transaction,
     },
-    rocksdb::{AsColumnFamilyRef, DB as RocksDb, IteratorMode, WriteBatchWithTransaction},
+    rocksdb::{AsColumnFamilyRef, DB as RocksDb, WriteBatchWithTransaction},
     std::{marker::PhantomData, sync::Arc},
     umi_blockchain::{
-        block::{BlockQueries, BlockRepository, BlockResponse, ExtendedBlock},
+        block::{BlockQueries, BlockRepository, BlockResponse, ExtendedBlock, ForkchoiceState},
         transaction::ExtendedTransaction,
     },
     umi_shared::primitives::B256,
 };
 
 pub const BLOCK_COLUMN_FAMILY: &str = "block";
 pub const HEIGHT_COLUMN_FAMILY: &str = "height";
+pub const FC_COLUMN_FAMILY: &str = "forkchoice";
```

Comment:
Is height database still needed? (the line above)

Reply:
Yes, the height column family is still needed because we still keep a height to hash mapping for when queries come with explicit heights (as opposed to a tag like `latest`). But this is now the only case where height is used because height is no longer the primary key for state lookup; height maps to a block hash (the mapping is forkchoice dependent) and the block hash is the primary key for looking up a state root.